utils/ttalps_plot_limits.py

```diff
@@ -1,129 +1,215 @@
 import ROOT
 from ttalps_cross_sections import cross_sections
 
-x_min = 1e-5 * 1e-3
-x_max = 1e5 * 1e-3
+year = "2018"
+cross_sections = get_cross_sections(year)
 
 y_min = 1e-3
-y_max = 1e3
-
-# output_name = "GoodBestLooseMuonsVertex_Pat_vxy"
+y_max = 1e4
+
+########  Signal Limits Summary Lxy Significance  ########
+####  PAT-PAT  ####
+    # ("tta_mAlp-0p35GeV_ctau-1e-5mm", 0.35): (53.9981,27.3901,37.3454,53.9375,79.0922,110.6069,),
+    # ("tta_mAlp-0p35GeV_ctau-1e0mm", 0.35): (12.4943,5.5664,8.1123,12.5000,19.4756,28.8879,),
+    # ("tta_mAlp-0p35GeV_ctau-1e1mm", 0.35): (55.5908,21.5596,33.5796,55.7500,91.7499,140.9686,),
+    # ("tta_mAlp-0p35GeV_ctau-1e2mm", 0.35): (558.9079,218.3594,338.1158,559.0000,919.9681,1425.6646),
+    # ("tta_mAlp-0p35GeV_ctau-1e3mm", 0.35): (4200.2824,1593.7500,2514.8926,4000.0000,7156.9458,11733.1719,),
+    # ("tta_mAlp-1GeV_ctau-1e-5mm", 1): (180.8130,101.6719,133.1796,180.7500,244.8725,317.4915,),
+    # ("tta_mAlp-1GeV_ctau-1e0mm", 1): (8.4522,3.7712,5.4961,8.4688,13.1609,19.5058,),
+    # ("tta_mAlp-1GeV_ctau-1e1mm", 1): (21.1509,7.9688,12.6379,21.2500,35.6496,55.3392,),
+    # ("tta_mAlp-1GeV_ctau-1e2mm", 1): (36.2018,13.5938,21.4703,36.2500,61.1030,95.2623,),
+    # ("tta_mAlp-1GeV_ctau-1e3mm", 1): (1344.4657,447.5781,750.0636,1348.0000,2422.6372,3981.1968,),
+    # ("tta_mAlp-2GeV_ctau-1e-5mm", 2): (162.8830,91.5469,119.9169,162.7500,220.4868,285.8741,),
+    # ("tta_mAlp-2GeV_ctau-1e0mm", 2): (8.3411,3.8604,5.5181,8.3750,12.8817,18.9778,),
+    # ("tta_mAlp-2GeV_ctau-1e1mm", 2): (12.7533,4.9048,7.6848,12.8125,21.1882,32.7127,),
+    # ("tta_mAlp-2GeV_ctau-1e2mm", 2): (78.5575,27.9932,45.4408,78.7500,135.8800,216.8075,),
+    # ("tta_mAlp-2GeV_ctau-1e3mm", 2): (570.3231,198.8594,324.2113,572.0000,1014.3232,1665.7311,),
+    # ("tta_mAlp-12GeV_ctau-1e0mm", 12): (47.9084,24.6855,33.5628,47.8750,69.0573,95.6612,),
+    # ("tta_mAlp-12GeV_ctau-1e1mm", 12): (12.5246,5.3489,7.9413,12.5625,19.9736,30.0839,),
+    # ("tta_mAlp-12GeV_ctau-1e2mm", 12): (24.1039,8.2930,13.6115,24.1250,42.9730,70.2744,),
+    # ("tta_mAlp-12GeV_ctau-1e3mm", 12): (),
+    # ("tta_mAlp-30GeV_ctau-1e0mm", 30): (),
+    # ("tta_mAlp-30GeV_ctau-1e1mm", 30): (),
+    # ("tta_mAlp-30GeV_ctau-1e2mm", 30): (),
+    # ("tta_mAlp-30GeV_ctau-1e3mm", 30): (),
+    # ("tta_mAlp-60GeV_ctau-1e0mm", 60): (82.0796,46.1250,60.4189,82.0000,111.0901,144.0349,),
+    # ("tta_mAlp-60GeV_ctau-1e1mm", 60): (),
+    # ("tta_mAlp-60GeV_ctau-1e2mm", 60): (32.2887,12.0142,19.0927,32.3750,54.8294,86.5208,),
+    # ("tta_mAlp-60GeV_ctau-1e3mm", 60): (),
+    # ("tta_mAlp-70GeV_ctau-1e0mm", 70): (150.7006,84.0684,110.5372,150.5000,203.8911,264.3567,),
```

Comment:
All this (and below) would be good to put in some text files with results, that we can keep track of, update, and use to make plots/tables.